changelogs/update-7319_enable_wc_pay_experiment

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: Enhancement
+
+Adds experiment for promoting wcpay in payment methods table. #7666
```

Comment:
Should we spell out WooCommerce Payments in this changelog entry?

Reply:
Also very nitpicky and something we aren't consistent on is verb tense in changelogs.